storm-core/src/jvm/backtype/storm/validation/ConfigValidation.java

```diff
@@ -0,0 +1,359 @@
+package backtype.storm.validation;
```

Comment:
Needs that Apache header.  Also not completely sure why we changed the package.  Not important though.